our-work/software/soma.yml

```diff
@@ -0,0 +1,14 @@
+title: SOMA pain & recovery
+description: A mobile app that combines symptom tracking with neuroscientific assessments and tailored interventions. SOMA is designed to directly support individuals on their journeys to overcome pain.
+languages:
+  - typescript
+  - react native
+tags:
+  - mobile
+people:
+  - Bradford Roarr
```

Comment:
The "person" map is missing the `name` key. Look at the schema for reference

Reply:
I added in the `name` key